apps/web/components/Organization/Wrapper/Themes/DefaultTheme/DefaultHeader.css.ts

```diff
@@ -1,60 +1,106 @@
 import { style } from '@vanilla-extract/css'
-import { blueberry100, themeUtils } from '@island.is/island-ui/theme'
+import { themeUtils } from '@island.is/island-ui/theme'
 
-export const headerBg = style({
-  height: 385,
-  marginTop: -130,
-  paddingTop: 130,
-})
-
-export const iconCircle = style({
-  height: 136,
-  width: 136,
-  margin: '0 auto',
-  display: 'flex',
-  justifyContent: 'center',
-  alignItems: 'center',
-  boxShadow: '0px 4px 30px rgba(0, 97, 255, 0.08)',
+export const gridContainer = style({
+  display: 'grid',
   ...themeUtils.responsiveStyle({
     xs: {
-      marginTop: 32,
+      gridTemplateRows: '200px 255px',
     },
-    md: {
-      marginTop: 104,
-      position: 'relative',
+    lg: {
+      gridTemplateRows: '255px',
+      gridTemplateColumns: '65fr 35fr',
     },
   }),
 })
 
-export const headerBorder = style({
+export const textContainer = style({
+  textAlign: 'center',
   ...themeUtils.responsiveStyle({
-    md: {
-      borderBottom: '4px solid #ffbe43',
+    xs: {
+      order: 1,
+      minHeight: '255px',
+      paddingTop: '32px',
+    },
+    lg: {
+      order: 0,
+      display: 'grid',
+      placeItems: 'center',
+      textAlign: 'left',
+      paddingTop: '0px',
     },
   }),
 })
 
-export const headerWrapper = style({
-  marginTop: -20,
-})
-
-export const headerLogo = style({
-  width: 70,
-  maxHeight: 70,
+export const textInnerContainer = style({
+  ...themeUtils.responsiveStyle({
+    lg: {
+      height: '100%',
+      maxWidth: '50%',
+      transform: 'translateX(50%)',
+      display: 'flex',
+      justifyContent: 'flex-end',
+      alignItems: 'center',
+      paddingLeft: '16px',
+      paddingRight: '16px',
+    },
+  }),
 })
 
-export const navigation = style({
+export const headerImage = style({
+  height: '100%',
+  width: '100%',
+  objectFit: 'cover',
   ...themeUtils.responsiveStyle({
-    md: {
-      background: 'none',
-      paddingTop: 0,
-    },
     xs: {
-      marginLeft: -24,
-      marginRight: -24,
-      paddingLeft: 24,
-      paddingRight: 24,
-      paddingTop: 32,
+      order: 0,
+    },
+    lg: {
+      order: 1,
     },
   }),
 })
+
+export const logoContainer = style({
+  width: '136px',
+  height: '136px',
+  boxShadow: '0px 4px 30px rgba(0, 97, 255, 0.08)',
+  position: 'absolute',
+  bottom: '-32px',
+  left: '7%',
+  display: 'grid',
+  placeItems: 'center',
+})
+
+export const logo = style({
+  width: '70px',
+  height: '70px',
+})
+
+export const contentContainer = style({
+  maxWidth: '1440px',
```

Comment:
Why is there a maxWidth?
Does it have something to with the images or is there a require style guide?
Just curious

Reply:
Ah, I understand. Thank you